src/main/java/core/basesyntax/service/csvfileservice/WriterService.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.csvfileservice;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriterService {
```

Comment:
you should create a separate interface with such a name and implement it here, right now it's a violation of the second principle of SOLID